omero/sysadmins/unix/server-installation.txt

```diff
@@ -248,22 +248,23 @@ Ice
 .. note::
 
     OMERO |version| supports 3.5 and 3.4 on UNIX and UNIX-like
-    platforms. You must install the correct version of OMERO.server
-    (see :downloads:`Downloads <>`).
+    platforms; support for 3.6 is currently upcoming. You must install
+    the correct version of OMERO.server (see :downloads:`Downloads
+    <>`).
 
 If possible, install one of the following packages:
 
-+-----------+---------------------------------+
-| System    | Package                         |
-+===========+=================================+
-| BSD Ports | devel/ice                       |
-+-----------+---------------------------------+
-| Debian    | zeroc-ice                       |
-+-----------+---------------------------------+
-| Homebrew  | ice (with --with-python option) |
-+-----------+---------------------------------+
-| RedHat    | N/A (install ZeroC RPMs)        |
-+-----------+---------------------------------+
++-----------+-------------------------------+
+| System    | Package                       |
++===========+===============================+
+| BSD Ports | devel/ice                     |
++-----------+-------------------------------+
+| Debian    | zeroc-ice                     |
++-----------+-------------------------------+
+| Homebrew  | ice (with --with-java option) |
```

Comment:
Don't you still need `--with-python`? And for 5.1should this be `zeroc-ice` from the `ome` tap?